src/python/CRABClient/JobType/PrivateMC.py

```diff
@@ -31,7 +31,7 @@ def run(self, *args, **kwargs):
             self.logger.debug("LHESource found in the CMSSW configuration.")
             configArguments['generator'] = getattr(self.config.JobType, 'generator', 'lhe')
 
-            major, minor, fix = os.environ['CMSSW_VERSION'].split('_')[1:4]
+            major, minor, _ = os.environ['CMSSW_VERSION'].split('_')[1:4]
```

Comment:
does it really make sense to use `_` as a variable ? I am surely one person who gets heavily confused by this. Is that some kind of PEP-8 standard for a variable which is never utilized ?